sample/HttpTrigger-CSharp-CustomRoute/function.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Could you add/modify a sample that shows using a docdb/table/etc input binding using {Id} to perform the lookup directly for you?